bip-0003.md

```diff
@@ -0,0 +1,694 @@
+one or more Shepherds to their BIP. Shepherds are stand-in owners of a BIP who were not involved in writing the
+document. They support the authors in advancing the proposal, or act as a point of contact for the BIP in the absence of the
+authors. Shepherds may perform the role of Authors for any aspect of the BIP process unless overruled by an Author.
+Shepherds share ownership of the BIP at the discretion of the Authors.
+
+### What is the significance of BIPs?
+
+BIPs do not define what Bitcoin is: individual BIPs do not represent Bitcoin community consensus or a general
+recommendation for implementation. A BIP represents a personal recommendation by the BIP authors to the Bitcoin
+community. Some BIPs may never be adopted. Some BIPs may be adopted by one or more Bitcoin clients or other related
+software. Some may even end up changing the consensus rules that the Bitcoin ecosystem jointly enforces.
+
+### What is the purpose of the BIPs repository?
+
+The [BIPs repository](https://github.com/bitcoin/bips/) serves as a publication medium and archive for mature proposals.
+Through its high visibility, it facilitates the community-wide consideration of BIPs and provides a well-established
+source to retrieve the latest version of any BIP. The repository transparently records all changes to each BIP and
+allows any community member to retain a complete copy of the archive easily.
+
+The BIPs repository is not a tool to track acceptance[^acceptance], adoption, or community consensus on BIPs, beyond
+providing a brief overview of BIP statuses (see [Workflow](#workflow) below) to the audience.
+There is no formal or informal decision body that governs Bitcoin development or decides acceptance of BIPs. Bitcoin
+development emerges from the participation of shareholders across the ecosystem.
+
+## BIP format and structure
+
+### Specification
+
+Authors may choose to submit BIPs in MediaWiki or Markdown[^markdown] format.
+
+Each BIP must have a _Preamble_, an _Abstract_, and a _Copyright_ section. Authors should consider all issues in the
+following list and address each as appropriate. For some BIPs, some concerns may not warrant a dedicated section.
+
+* Preamble -- Headers containing metadata about the BIP (see the section [BIP header preamble](#bip-header-preamble)
+  below).
+* Abstract -- A short description of the technical issue being addressed.
+* Motivation -- The motivation is critical for BIPs. It should clearly explain what issue the BIP addresses, and how the
+  existing situation is inadequate to address the problem that the BIP solves.
+* Specification -- The technical specification should describe the syntax and semantics of any new feature. The
+  specification should be detailed enough to enable any Bitcoin project to create an interoperable implementation.
+* Rationale -- The rationale fleshes out the specification by describing what motivated the design and why particular
```

Comment:
maybe avoid the word "motivated" here as it is also a section title

Reply:
Slightly amended this sentence.